client/spec.yaml

```diff
@@ -495,4 +495,178 @@ paths:
           description: Bad request
         "5XX":
           description: Server side error while tailing logs
-
+  /api/logs/v1/{tenant}/loki/api/v1/rules:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+    get:
+      tags:
+        - logs/rulesv1
+      summary: Get Logs Rules for a tenant in JSON form
+      operationId: getAllLogsRules
+      description: |
+        You can get Logs rules for a tenant with health data
+      responses:
+        "2XX":
+          description: All log rules for a tenant
+          content:
+            application/json:
+              schema:
+                $ref: "./responses/responses.yaml#/components/schemas/AllLogRulesResponse"
+        "4XX":
+          description: Bad request
+        "5XX":
+          description: Server side error
+  /api/logs/v1/{tenant}/loki/api/v1/rules/{namespace}:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+      - $ref: ./parameters/parameters.yaml#/components/parameters/logRulesNamespace
+    get:
+      tags:
+        - logs/rulesv1
+      summary: Get Logs Rules for a tenant in JSON form inside a namespace
+      operationId: getLogsRules
+      description: |
+        You can get Logs rules for a tenant in a namespace with health data
+      responses:
+        "2XX":
+          description: rule for a tenant
```

Comment:
I think this description and the remaining ones in this file can be updated too to fit more than 1 rule